src/coreclr/vm/field.h

```diff
@@ -39,7 +39,7 @@ class FieldDesc
 #endif
 
   protected:
-    RelativePointer<PTR_MethodTable> m_pMTOfEnclosingClass;  // This is used to hold the log2 of the field size temporarily during class loading.  Yuck.
+    PTR_MethodTable m_pMTOfEnclosingClass;  // This is used to hold the log2 of the field size temporarily during class loading.  Yuck.
```

Comment:
In a future cleanup PR, we should make this field a union to make the "Yuck" comment a little more formalized so we aren't just storing an integer in a "dac-ized" pointer field.